lib/logstash/filters/translate.rb

```diff
@@ -53,20 +56,23 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
   # NOTE: it is an error to specify both `dictionary` and `dictionary_path`
   config :dictionary, :validate => :hash,  :default => {}
 
-  # The full path of the external YAML dictionary file. The format of the table
-  # should be a standard YAML file. Make sure you specify any integer-based keys
-  # in quotes. The YAML file should look something like this:
+  # The full path of the external dictionary file. The format of the table
+  # should be a standard YAML, JSON or CSV. Make sure you specify any integer-based keys
+  # in quotes. For example, the YAML file should look something like this:
   # [source,ruby]
   #     "100": Continue
   #     "101": Switching Protocols
   #     merci: gracias
   #     old version: new version
-  #     
+  # 
   # NOTE: it is an error to specify both `dictionary` and `dictionary_path`
+  # NOTE: Currently supported formats are YAML, JSON and CSV, format selection is
+  # based on the file extension, json for json, (yaml|yml) for YAML and csv for CSV.
+  # NOTE: Important to use equivalent formtats for JSON and CSV, only simple hash are used for now.
```

Comment:
Should be rewritten for clarity: "The JSON format only supports simple key/value, unnested objects. The CSV format expects exactly two columns with the first serving as the original text, the second column as the replacement.